lib/rzo/app/templates/environment.conf.erb

```diff
@@ -0,0 +1,18 @@
+# Each environment can have an environment.conf file. Its settings will only
```

Comment:
perhaps it should just do this within the puppetmaster, so `/etc/puppetlabs/code/environment.conf` or wherever it is on the host.